README.md

```diff
@@ -48,36 +48,32 @@ Examples of Data broker platforms are ARC, Galaxy, ...
+This command line tool (CLI) will be used by the Data broker platform and will perform the actual submission of the ISA-JSON to the repositories. Based on receipts repositories give back as response, the ISA-JSON will be updated with accession numbers. The application is build as a Python library which can be integrated in a web application, ARC, Galaxy and others. Source code and documentation can be found in the [mars-cli folder](/mars-cli/) in this repo.
 
 The main steps of MARS-CLI are:
 
-1. **Validating the ISA-JSON**: Syntax validation
+1. **Ingesting and validating the ISA-JSON**: Compared to the vanilla ISA specification, the MARS-CLI has certain fields required (for example `targetRepository`) in order to function properly. Upon ISA-JSON ingestion the information gets loaded in the
 
-    => We could use the [ISA-API validation](https://isa-tools.org/isa-api/content/validation.html) library.
+2. **Identifying the target repositories**: The order of submission can be depended on the target repositories specified in the ISA-JSON.
 
-2. **Registering samples in BioSamples**: Submitting an ISA-JSON to a newly developed API at BioSamples. The BioSamples accession will be reused by the other repositories and thus needs to be done first.
-After a successful submission, BioSamples sends back an updated ISA-JSON containing BioSamples accession numbers for `Source` and `Sample` as `Source characteristics` and `Sample characteristics`, respectively. 
+3. **Registering samples in BioSamples**: Submitting an ISA-JSON to a newly developed API at BioSamples. The BioSamples accession will be reused by the other repositories and thus needs to be done first.
+After a successful submission, BioSamples sends back a receipt containing BioSamples accession numbers for `Source` and `Sample` as `Source characteristics` and `Sample characteristics`, respectively. 
 
-    => The source code for the ISA-JSON API for BioSamples can be found in the [repository-services repo](/repository-services/isajson-biosamples/) and can be used for testing
+    > The source code for the ISA-JSON API for BioSamples can be found in the [repository-services repo](/repository-services/isajson-biosamples/) and can be used for testing
 
-3. **Filtering the ISA-JSON**: The ISA-JSON (updated with BioSamples IDs) has to be filtered for every target repository so it only contains information relevant for that repo
+4. **Filtering the ISA-JSON**: The ISA-JSON (updated with BioSamples IDs) has to be filtered for every target repository so it only contains information relevant for that repo. This will be facilitated by the `targetRepository` attribute present in the ISA-JSON assays.
```

Comment:
The comment for the target repository looks like this in [ISA-BH2023-ALL](https://github.com/elixir-europe/MARS/blob/main/test-data/ISA-BH2023-ALL/isa-bh2023-all.json):
```json
{
    "name": "target repository",
    "value": "metabolights"
}
```
So should it be this?

```suggestion
4. **Filtering the ISA-JSON**: The ISA-JSON (updated with BioSamples IDs) has to be filtered for every target repository so it only contains information relevant for that repo. This will be facilitated by the `target repository` attribute present in the ISA-JSON assays.
```